### ISSUE TITLE
Set Up ADFS for Signs & Markings app

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#274]

**Comment by johnclary** [comment]
Reached out to CTM. My email below. Going to do this one along with VZA and Banners.

Hi John and Michael—hope the cert update went smoothly for you. No issues on our side!

I have three new apps that I need to setup with ADFS validation. Do y'all have bandwidth to do this in the next three weeks?


Checking my notes, I believe I just need to send you our metadata and claim rules. Anything else I'm missing?


I'll kick these off in separate threads. Should I open HD tickets?


Thanks.

John

**Comment by johnclary** [comment]
HD ticket submitted. Going to try with the new Azure AD services. Am told the level of effort is the same but it doesn't involve on-prem servers, so it should be more reliable.

**Comment by johnclary** [comment]
I've got everything configured and sent our metdata to John D. Email below.

I also updated [our wiki docs](https://github.com/cityofaustin/atd-data-tech/wiki/Knack:-SSO---ADFS-Setup) with instructions about how we managed our certs.

>Ok, I've got things configured on our end. I have attached our metadata and pasted below.
>
One possible issue is that we were previously using the windowsaccountname claim as our NameIDFormat. >But that claim is not in the new metadata. So i just used emailAddress.
>
>We'll see how that goes. Let me know when you're ready to test.
